USFMToolsSharp/HtmlRenderer.cs

```diff
@@ -286,6 +286,16 @@ private string RenderMarker(Marker input)
                     }
                     output.AppendLine("</span>");
                     break;
+                case FQMarker fqMarker:
+                    output.Append("<span class=\"footnote-alternate-translation\">");
+                    foreach (Marker marker in input.Contents)
+                    {
+                        output.Append(RenderMarker(marker));
+                    }
+                    break;
+                case FQEndMarker fqEndMarker:
+                    output.Append("</span>");

```

Comment:
This should be taken care of all in the fq render. The end markers aren't actually rendered out

Reply:
@rbnswartz Yeah I also fixed the FQAMarker